_posts/2018-12-06-pmreorder-basics.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Pmreorder basics
+author: wlemkows
+layout: post
+identifier: pmreorder-basics
+---
+
+
+### Introduction
+
+
+When you write your persistent memory application the good practice is
```

Comment:
It's good practice to run persistent memory applications under pmemcheck - a tool which is described [here](#link) and [here](#link2).